GitCommands/Settings/AppSettings.cs

```diff
@@ -90,6 +90,12 @@ static AppSettings()
             }
         }
 
+        public static bool TelemetryEnabled
+        {
+            get => GetBool("TelemetryEnabled", true);
```

Comment:
We could add another Setting like `TelemetryApproved` in oder to ask the user on the first start of GE.
```suggestion
            get => GetBool("TelemetryEnabled", false);
```

Reply:
I'm planning something better
![image](https://user-images.githubusercontent.com/4403806/57212988-c9505a80-7028-11e9-816d-0b1e26a88335.png)

Need to work out how to execute a custom action at the end of the install.

Reply:
I like this idea. The installer is the right place to inform about telemetry and giving the option to opt-out. In this case I also agree with having the checkbox selected by default (i.e. _opt-out_, not _opt-in_), because it is well visible.

Reply:
Yes, that's less intrusive than a nag screen. But the portable version will still need something else.

Reply:
> But the portable version will still need something else.

Bah! Yes, you're right 😒 
Any thoughts?

Reply:
I made the button more prominent, perhaps it is sufficient.
It is not visible in the dashboard though...

![image](https://user-images.githubusercontent.com/4403806/57489266-d0ad8780-72f8-11e9-8e65-d7c1b4934e9c.png)

Reply:
The default should still be off, use opt-in

Reply:
Respectfully disagree, if we have it disabled by default no one will turn it on. It is just a fact of life.

However if we have it on by default, those who actively dislike tracking (I'm one of those) will go out of their way and turn if off.